site/en/blog/extension-safety-hub/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Bringing Safety Check to the chrome://extensions page"
+description: "Starting in Chrome 117, Chrome will proactively flag to users when an extension they have installed is no longer in the Chrome Web Store."  
+layout: "layouts/blog-post.njk"
+authors:
+  - oliverdunk
+date: 2023-08-16
+tags:
+  - extensions-news
+---
+
+Starting in Chrome 117, Chrome will proactively flag to users when an extension they have installed is no longer in the Chrome Web Store. This is limited to three specific cases:
+
+- The extension has been unpublished by the developer.
+- The extension has been taken down for violating Chrome Web Store policy.
+- The item was marked as malware.
+
+If an issue has been resolved, the notification will be automatically cleared. The notification will not be displayed for an extension which has been [notified of a possible violation][violation-notification] and has been given time to address the issue or appeal.
+
+For most users, they are most likely to encounter this feature in the "Privacy and security" section of the settings page.
+
+<figure>
+  {% Img src="image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/QgVuPAOzxISY05yD7cND.png", alt="Chrome will highlight these extensions in the Privacy and security settings", width="800", height="537", class="screenshot" %}
+  <figcaption>
+  Chrome will highlight these extensions in the Privacy and security settings
+  </figcaption>
+</figure>
+
+When clicking "Review", they will be taken to their extensions and given the choice to either remove the extension or hide the warning if they wish to keep the extension installed. As in previous versions of Chrome, extensions marked as malware are automatically disabled.
```

Comment:
This needs a transition to indicate that you stopped talking about users and started talking about developers.

Reply:
I'm not sure I follow. This is still talking about things a user can do.

Reply:
'review' and 'their extensions' made me think you had switched contexts. I made a suggestion that clarifies the matter. (I think I would have made this suggestion even if I hadn't misinterpreted the sentence.)